cli/dcos-spark/submit_builder.go

```diff
@@ -146,6 +147,10 @@ Args:
 	submit.Flag("tgt-secret-value", "Value of TGT to be used in the drivers, must be base64 encoded").
 		Default("").StringVar(&args.tgtSecretValue)
 
+	// TODO expose krb5 config (base64)
```

Comment:
probably can't hurt to update the docs at L78-L116 to reflect the current state of things